djcelery/__init__.py

```diff
@@ -6,7 +6,7 @@
 import os
 import sys
 
-VERSION = (3, 2, 0)
+VERSION = (3, 2, 2)
```

Comment:
I'm sure this would work, but we should consider using a proper [local version identifier](https://www.python.org/dev/peps/pep-0440/#local-version-identifiers).  Something like "3.2.1+edx.1" would make it clear that this is our fork of a particular upstream release, and this format is supported by all recent pip and setuptools versions.  Avoids confusion if upstream actually decides to release a 3.2.2 someday.